.cspell/cspell.config.yml

```diff
@@ -133,3 +133,5 @@ import:
   - "@cspell/dict-bash/cspell-ext.json"
   - "@cspell/dict-shell/cspell-ext.json"
   - "@cspell/dict-win32/cspell-ext.json"
+enableFiletypes:
```

Comment:
Is this only needed for actions files ?

[the documentation uses json format as example](https://streetsidesoftware.com/vscode-spell-checker/docs/configuration/#cspellenablefiletypes), I don't understand why we didn't need it before (and why now we don't have to also set all the other types we want checked) :thinking:

Reply:
I now known understand the culprit: 

it's `allowCompoundWords == true` with the current setting, it allow `accross` because it's a combination of `acc+ross` :exploding_head: